packages/gallery/src/components/item/itemHelper.js

```diff
@@ -0,0 +1,73 @@
+import { window, utils, isSiteMode, isSEOMode } from 'pro-gallery-lib';
+
+function shouldChangeActiveElement() {
+  return (isSiteMode() || isSEOMode()) && !utils.isMobile() && window.document;
+}
+
+export function onAnchorFocus({
+  itemContainer,
+  shouldUseExperimentalFeature,
+  itemAnchor,
+}) {
+  if (shouldChangeActiveElement() && shouldUseExperimentalFeature) {
+    /* Relevant only for Screen-Reader cases:
+         When we navigate on the accessibility tree, screen readers stops and focuses on the <a> tag,
+         so it will not go deeper to the item-container keyDown event */
+    const activeElement = window.document.activeElement;
+    if (activeElement === itemAnchor) {
+      itemContainer.focus();
+    }
+  }
+}
+
+export function changeActiveElementIfNeeded({
+  prevProps,
+  currentProps,
+  itemContainer,
+}) {
+  try {
+    if (
+      shouldChangeActiveElement() &&
+      window.document.activeElement.className
+    ) {
+      const activeElement = window.document.activeElement;
+
+      //check if focus is on 'gallery-item-container' in current gallery
+      const isThisGalleryItemInFocus = () =>
```

Comment:
take this function outside and make the argument be (activeElement) or make it take the active element on its own. it doesn't need to be a part of the scope here.